server/routes/wazuh-utils.ts

```diff
@@ -47,4 +47,26 @@ export function WazuhUtilsRoutes(router: IRouter) {
     },
     async (context, request, response) => ctrl.getAppLogs(context,request, response)
   );
+
+  router.get(
+    {
+      path: '/utils/logs/frontend',
+      validate: false
+    },
+    async (context, request, response) => ctrl.getFrontendLogs(context,request, response)
+  );
+
+  router.put(
+    {
+      path: '/utils/logs/frontend',
+      validate: {
+        body: schema.object({
+          message: schema.string(),
+          level: schema.string(),
+          location: schema.string()
+        })
+      }
+    },
+    async (context, request, response) => ctrl.updateFrontendLogs(context, request, response)
+  );
```

Comment:
same here move these routes to a new file into the new folder `wazuh-utils` and export it from an index